source/index.html.md

```diff
@@ -261,6 +264,77 @@ turnaroundTime (OPTIONAL) | `FIVE_DAYS` | `ONE_DAY`, `TWO_DAYS`, `THREE_DAYS`, `
 
 _If you need support for a different file format, please get in touch with us: info (at) amberscript (dot) com_
 
+## Requesting for translated subtitles
```

Comment:
Please change it to: "Request for translated subtitles for an existing manual captions job"